src/Features/Blockcore.Features.ColdStaking/UI/Pages/ColdStakeReceive.razor

```diff
@@ -0,0 +1,218 @@
+
+@{
+    <div class="d-flex justify-content-between flex-wrap flex-md-nowrap align-items-center pb-2 mb-3 border-bottom">
+         <h1 class="h2"><strong>@walletname : @accountType : receive </strong></h1>
+        <div class="btn-toolbar mb-2 mb-md-0">
+             <button class="btn btn-sm btn-primary mr-1" @onclick="() => { NavigateToColdStakeView(walletname, accountname); }">
+                <span class="oi oi-pulse" aria-hidden="true"></span> ColdStake View
+            </button>
+        </div>
+    </div>
+
+    var result = this.WalletManager.GetUnusedAddress(new WalletAccountReference(walletname, accountname));
+
+    <div class="card mb-3">
+        <div class="card-header">
+            <div class="card-title">Address</div>
+        </div>
+        <div class="card-body">
+            <div class="row">
+                <div class="col-sm-2"><strong>Address:</strong></div>
+                <div class="col-sm-10"><CopyToClipboard Text="@result.Address" /></div>
+            </div>
+        </div>
+    </div>
+
+    Blockcore.Features.Wallet.Types.Wallet wallet = this.WalletManager.GetWallet(walletname);
+    HdAccount account = wallet.GetAccount(accountname);
+
+    var accountmodel = new AddressesModel
+            {
+                Addresses = account.GetCombinedAddresses().Select(address =>
+                {
+                    (Money confirmedAmount, Money unConfirmedAmount, bool anyTrx) = address.GetBalances(wallet.walletStore, account.IsNormalAccount());
+
+                    return new AddressModel
+                    {
+                        // Address = (bool)(NodeDeployments.GetFlags().ScriptFlags.HasFlag(ScriptVerify.Witness)) ? address.Bech32Address : address.Address,
+                        Address = address.Address,
+                        IsUsed = anyTrx,
+                        IsChange = address.IsChangeAddress(),
+                        AmountConfirmed = confirmedAmount,
+                        AmountUnconfirmed = unConfirmedAmount
+                    };
+                })
+            };
+
+    <div class="row mb-3">
+        <div class="col-12 grid-margin">
+            <div class="card">
+                <div class="card-header d-flex justify-content-between align-items-center">
+                    <h5 class="card-title">
+                        Account Address
+                    </h5>
+                    <div class="btn-group" role="group" aria-label="Basic example">
+                                       
+                    @if (_ShowTab_UnUsedAddress)
+                    {
+                        <button type="button" class="btn btn-sm btn-primary mr-1" @onclick="generateAddress">Generate a new address</button>
+                    }
+
+                        <button type="button" class="btn btn-sm btn-primary " @onclick="tabShowUsedAddress">Used</button>
+                        <button type="button" class="btn btn-sm btn-primary " @onclick="tabShowUnUsedAddress">UnUsed</button>
```

Comment:
"Unused" is not two words I think?

Reply:
No it can be like this, unless you mean to make the second U lower case?